### PR TITLE
DSG-2765: Public dashboard alert formatting

### PR DESCRIPTION
**Pull request by sunnyjiao-nv**
# Basics

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [x] Styling and layout changes
- [ ] Code style changes (formatting, local variables)
- [ ] Refactoring (no functional changes)
- [ ] Build or CI related changes
- [ ] Tests
- [ ] Other... Please describe:

## What changes does this PR introduce?
- Fixes the document alert icon formatting, as well as some margins & font sizes in the Dashboard to match Figma

## Screenshot of the affected changes
Before:
Screenshot 2024-01-25 at 12 24 40 PM

After
Screenshot 2024-01-25 at 12 17 56 PM

## Checklist

- [ ] I have updated the documentation accordingly
- [ ] I have confirmed all acceptance criteria defined by the story have been met
- [ ] I have confirmed that Sonar Qube findings for new code are reviewed and fixed, or will be whitelisted

### CODE REVIEW

File: libs/public/feature/dashboard/src/lib/components/dashboard/dashboard.component.scss [hunk 1]
```diff
@@ -155,32 +157,32 @@
...
 
-    ::ng-deep {
       nuverial-icon {
-        margin: 0 0.1rem 0 -0.5rem;
-      }
-      nuverial-icon mat-icon {
-        color: var(--theme-color-priority-urgent) !important;
+        margin: 0 0.4rem 0 -0.1rem;
+        ::ng-deep mat-icon {
+          color: var(--theme-color-priority-urgent) !important;
```
Comment (davidyoung77) [inline libs/public/feature/dashboard/src/lib/components/dashboard/dashboard.component.scss:174]: A better way to do this is by setting these 2 css vars
`--nuverial-icon-size`, `-nuverial-icon-color`, please see the icon style sheet, most of our nuverial components have exposed this type of styling overridability for ease of use

Comment (sunnyjiao-nv) [reply libs/public/feature/dashboard/src/lib/components/dashboard/dashboard.component.scss]: Done - made that change for the empty transactions CTA icon as well

File: libs/public/feature/dashboard/src/lib/components/dashboard/dashboard.component.scss [hunk 2]
```diff
@@ -125,6 +118,7 @@
     }
 
     .application-transaction-id {
+      font-size: rem.fromPx(18px);
```
Comment (thechechogomez) [inline libs/public/feature/dashboard/src/lib/components/dashboard/dashboard.component.scss:121]: This font-sizes should be variables. 

```suggestion
      font-size: var(--theme-typography-font-size-large);
```

File: libs/public/feature/dashboard/src/lib/components/dashboard/dashboard.component.scss [hunk 3]
```diff
@@ -138,6 +132,7 @@
     }
 
     .application-status {
+      font-size: rem.fromPx(18px);
```
Comment (thechechogomez) [inline libs/public/feature/dashboard/src/lib/components/dashboard/dashboard.component.scss:135]:
```suggestion
      font-size: var(--theme-typography-font-size-large);
```

File: libs/public/feature/dashboard/src/lib/components/dashboard/dashboard.component.scss [hunk 4]
```diff
@@ -155,32 +150,28 @@
...
-    color: var(--theme-color-priority-urgent) !important;
-    display: flex;
+  .documents-requested {
+    font-weight: bold;
 
-    span {
-      padding-top: 0.3rem;
-    }
+    .alert {
+      font-size: rem.fromPx(14px);
```
Comment (thechechogomez) [inline libs/public/feature/dashboard/src/lib/components/dashboard/dashboard.component.scss:158]:
```suggestion
      font-size: var(--theme-typography-font-size-small);
```

File: libs/public/feature/dashboard/src/lib/components/dashboard/dashboard.component.scss [hunk 5]
```diff
@@ -155,32 +150,28 @@
...
+        --nuverial-icon-size: 1.5rem;
       }
     }
-  }
 
-  ul {
-    padding-left: 1rem;
-    margin: 0.3rem 0;
+    ul {
+      font-size: rem.fromPx(16px);
```
Comment (thechechogomez) [inline libs/public/feature/dashboard/src/lib/components/dashboard/dashboard.component.scss:172]:
```suggestion
      font-size: var(--theme-typography-font-size-normal);
```

**Review by thechechogomez — APPROVED** [review]
Looks good. Just a few suggestions

**Review by thechechogomez — APPROVED** [review]

**Pull request merged**